### ISSUE TITLE
两处注释错误:

### ISSUE THREAD

**Issue by gengxuelong** [wenet-e2e/wenet#2006]
1. wenet/transducer/predictor.py中:
def ApplyPadding(input, padding, pad_value) -> torch.Tensor:
    """
    Args:
        input:   [bs, max_time_step, dim]         bug:================> dim不应存在,
        padding: [bs, max_time_step]
    """
    return padding * pad_value + input * (1 - padding)

2. wenet/utils/common.py中: 
def pad_list(xs: List[torch.Tensor], pad_value: int):
    """Perform padding for the list of tensors.

    Args:
        xs (List): List of Tensors [(T_1, `*`), (T_2, `*`), ..., (T_B, `*`)].    bug:=================>'*'不应存在,或者说代码未实现 * 
        pad_value (float): Value for padding.

    Returns:
        Tensor: Padded tensor (B, Tmax, `*`).                             bug: ===================>同上

    Examples:
        >>> x = [torch.ones(4), torch.ones(2), torch.ones(1)]
        >>> x
        [tensor([1., 1., 1., 1.]), tensor([1., 1.]), tensor([1.])]
        >>> pad_list(x, 0)
        tensor([[1., 1., 1., 1.],
                [1., 1., 0., 0.],
                [1., 0., 0., 0.]])

    """
    n_batch = len(xs)
    max_len = max([x.size(0) for x in xs])
    pad = torch.zeros(n_batch, max_len, dtype=xs[0].dtype, device=xs[0].device)
    pad = pad.fill_(pad_value)
    for i in range(n_batch):
        pad[i, :xs[i].size(0)] = xs[i]

    return pad

**Comment by xingchensong** [comment]
https://github.com/wenet-e2e/wenet/pull/2009
